lib/backends/webgl/webgl-context.ts

```diff
@@ -103,7 +103,7 @@ export class WebGLContext {
   readTexture(
       texture: WebGLTexture, width: number, height: number, dataSize: number, dataType: Encoder.DataType,
       channels: number): Encoder.DataArrayType {
-    const gl = this.gl;
+    const gl = this.gl as WebGL2RenderingContext;
```

Comment:
>const gl = this.gl as WebGL2RenderingContext [](start = 4, length = 44)

Looks like this branch is based off the old pack_share branch. There are a handful changes made to that branch afterwards before merging. Maybe it's not a bad idea to try rebase.

Reply:
fixed.